nodes/textures/wireframe.py

```diff
@@ -0,0 +1,34 @@
+import bpy
+from bpy.props import EnumProperty, FloatProperty
+from ..base import LuxCoreNodeTexture
+from ... import utils
+from ...utils import node as utils_node
+
+
+class LuxCoreNodeTexWireframe(bpy.types.Node, LuxCoreNodeTexture):
+    bl_label = "Wireframe"
+    bl_width_default = 200
+
+    #width: FloatProperty(update=utils_node.force_viewport_update, name="Line Width", description="Width of line", min=0, subtype="DISTANCE", unit="LENGTH", default=0.1)
+    
+    def init(self, context):
+        self.add_input("LuxCoreSocketFloatPositive", "width", 0.1)
+        self.add_input("LuxCoreSocketColor", "border", (0.7, 0.7, 0.7))
+        self.add_input("LuxCoreSocketColor", "inside", (0.2, 0.2, 0.2))
```

Comment:
All names used in the UI should be capitalized: "Width" instead of "width", "Border" instead of "border", etc.